lib/src/core/parameter/mixins/language_parameter.dart

```diff
@@ -0,0 +1,9 @@
+import '../../util/strings.dart' as strings;
+
+mixin LanguageParameter {
```

Comment:
You can create only one ParameterClass, with diferrent kind of get depending of langParam, pageParam, etc